test/lit.cfg

```diff
@@ -633,6 +633,8 @@ if run_vendor == 'apple':
     )
 
     config.available_features.add('objc_interop')
+    config.available_features.add('dispatch')
```

Comment:
Can you use libdispatch instead of dispatch? That way it is really clear what this is for?